src/clr-angular/popover/dropdown/providers/dropdown-focus-handler.service.ts

```diff
@@ -92,11 +83,10 @@ export class DropdownFocusHandler implements FocusableItem {
   set trigger(el: HTMLElement) {
     this._trigger = el;
     this.renderer.setAttribute(el, 'id', this.id);
+
     if (this.parent) {
-      // The root trigger needs to be in the tab flow, but nested ones are removed like any other dropdown item.
-      this.renderer.setAttribute(el, 'tabindex', '-1');
+      this.renderer.listen(el, 'keydown.arrowright', event => this.ifOpenService.toggleWithEvent(event));
```

Comment:
We need to remove these listeners to prevent DOM leakages.

Reply:
Added unlisteners to this service and `focus-service.ts` class. This class's onDestroy method will call both.